### PR TITLE
Feature/entries in analysis module

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/views/PillarAnalysis/EntriesFilterForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+                variant="tertiary"
+            >
+                {_ts('leads', 'filterClearFilter')}
+            </Button>
+            <Button
+                className={styles.button}
+                onClick={allFiltersVisible ? hideAllFilters : showAllFilters}
+                variant="tertiary"
+            >
+                {allFiltersVisible ? 'Hide filters' : 'Show all filters'}
```
Comment (samshara) [inline src/views/PillarAnalysis/EntriesFilterForm/index.tsx]: use string translations

File: src/views/PillarAnalysis/index.tsx [hunk 2]
```diff
@@ -52,6 +77,13 @@ function PillarAnalysis(props: PageProps) {
     } = props;
 
     const [value, setValue] = useState<string | undefined>();
+    const [activeTab, setActiveTab] = useState<TabNames>('entries');
+    const [filtersValue, setFiltersValue] = useState<FaramValues>({});
+    const [entries, setEntries] = useState<Partial<EntryFields>[]>([]);
```
Comment (samshara) [inline src/views/PillarAnalysis/index.tsx]: why are we using Partial here?

File: src/views/PillarAnalysis/index.tsx [hunk 3]
```diff
@@ -63,6 +95,99 @@ function PillarAnalysis(props: PageProps) {
...
+        geoOptions,
+    ] = useRequest<GeoOptions>({
+        url: 'server://geo-options/',
+        method: 'GET',
+        query: geoOptionsRequestQueryParams,
+        schemaName: 'geoOptions',
+        autoTrigger: true,
+        onSuccess: (response) => {
+            console.warn('here', response);
+        },
```
Comment (samshara) [inline src/views/PillarAnalysis/index.tsx]: remove console.warn()

File: src/views/PillarAnalysis/index.tsx [hunk 4]
```diff
@@ -90,7 +215,7 @@ function PillarAnalysis(props: PageProps) {
                 {breadcrumb(pillarAnalysis?.analysisName, pillarAnalysis?.title ?? '')}
```
Comment (samshara) [inline src/views/PillarAnalysis/index.tsx:217]:
```suggestion
              {breadcrumb(pillarAnalysis && pillarAnalysis.analysisName, pillarAnalysis?.title ?? '')}
```

**Review by subinasr — APPROVED** [review]

File: src/views/PillarAnalysis/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply src/views/PillarAnalysis/index.tsx]: I've only requested items required using queries. Is this a bad approach?

Comment (frozenhelium) [reply src/views/PillarAnalysis/index.tsx]: Maybe use pick? 
CC @tnagorra

**Review by frozenhelium — APPROVED** [review]

**Review by tnagorra — APPROVED** [review]

**Review by subinasr — APPROVED** [review]

**Pull request merged**